packages/web/app/src/components/target/settings/cdn-access-tokens.tsx

```diff
@@ -396,20 +383,18 @@ export function CDNAccessTokens(props: {
           </>
         }
       />
-      {canManage && (
-        <div className="my-3.5 flex justify-between">
-          <Button asChild>
-            <Link
-              search={{
-                page: 'cdn',
-                cdn: 'create',
-              }}
-            >
-              Create new CDN token
-            </Link>
-          </Button>
-        </div>
-      )}
+      <div className="my-3.5 flex justify-between">
```

Comment:
so if user doesn't have access to manage CDN then they won't be able to access this page?

Reply:
Yes, there is no point of seeing CDN access tokens if you cannot modify them.

Reply:
if necessary continue discussion in https://github.com/graphql-hive/console/pull/6327